src/index.html

```diff
@@ -1,18 +1,118 @@
 <!doctype html>
 <html lang="en">
+  <!-- #region head -->
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__cord-image">
+        <img
+          src="/src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27” Retina 5K Monoblock"
+          class="card__image"
+        />
+      </div>
+
+      <div class="card__info-product">
+        <h3 class="card__title-product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <span class="card__article-product">Product code: 195434</span>
+      </div>
+
+      <div class="card__rating">
+        <div class="card__rating-stars star">
+          <div class="star__line star__line--0">
+            <div class="star__style star__style--passive"></div>
+            <div class="star__style star__style--passive"></div>
+            <div class="star__style star__style--passive"></div>
+            <div class="star__style star__style--passive"></div>
+            <div class="star__style star__style--passive"></div>
+          </div>
```

Comment:
Fix classes and implement the modifier logic
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```